custom_components/lennoxs30/__init__.py

```diff
@@ -92,16 +132,16 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
         else:
             poll_interval = DEFAULT_LOCAL_POLL_INTERVAL
 
-    t = config.get(DOMAIN).get(CONF_FAST_POLL_INTERVAL)
+    t = entry.data[CONF_FAST_POLL_INTERVAL]
     if t != None and t > 0.2:
         fast_poll_interval = t
     else:
         fast_poll_interval = DEFAULT_FAST_POLL_INTERVAL
 
-    allergenDefenderSwitch = config.get(DOMAIN).get(CONF_ALLERGEN_DEFENDER_SWITCH)
-    app_id = config.get(DOMAIN).get(CONF_APP_ID)
-    conf_init_wait_time = config.get(DOMAIN).get(CONF_INIT_WAIT_TIME)
-    create_sensors = config.get(DOMAIN).get(CONF_CREATE_SENSORS)
+    allergenDefenderSwitch =  entry.data[CONF_ALLERGEN_DEFENDER_SWITCH]
+    app_id =  entry.data[CONF_APP_ID]
+    conf_init_wait_time = entry.data[CONF_INIT_WAIT_TIME]
+    create_sensors =  entry.data[CONF_CREATE_SENSORS]
     create_inverter_power = config.get(DOMAIN).get(CONF_CREATE_INVERTER_POWER)
```

Comment:
This should be looking at entry data

  create_inverter_power = entry.data[CONF_CREATE_INVERTER_POWER]

Reply:
Looks like the last minute merge got me there. I'll update my branch this morning but won't be able to test it.